content/faq/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Text-to-Speech on Android
+description: Guide on how to make TTS work on Android
+weight: 100
+extra:
+    menu_title: FAQ
+---
+
+TLDR:
+
+Organic Maps uses the system text-to-speech (TTS) engine for voice instructions. The default engines vary by device. The choices can include Google Text-to Speech, device manufacturer's engine or a third-party one. 
+
+You can use a third-party Text-to-speech engine which can be downloaded from Google Play or F-Droid. 
+
+[RHVoice](https://rhvoice.org/) is a free and open source speech engine which can be downloaded from [Google Play](https://play.google.com/store/apps/details?id=com.github.olga_yakovleva.rhvoice.android) and [F-Droid](https://f-droid.org/en/packages/com.github.olga_yakovleva.rhvoice.android/).
+
+## Instructions
+
+- Open the Settings app on your Android device.
+- Select Additional Settings and then select Accessibility.
+- Choose your preferred engine, speech rate and pitch.
+
+If you cannot find the relevant setting, open the settings app and search for Text-to-speech.
+
+P.S: Do note that these steps may vary based on the brand you are using.
```

Comment:
It would be great to mention this list of other TTS engines. And even better, to port it here as a separate maintainable markdown table, like a crypto table.
https://docs.google.com/spreadsheets/d/1kZ6cKY_BcCkxmGtCioQi9Z0SvhcWBkf427DQCWSwIDs/edit#gid=0

Reply:
Done! Will be pushing to the branch.